### PR TITLE
fix: avatar cutting off on contribution bar

### LINKED ISSUE #632: Prevent avatars in mining progress bar from being cut off
Here's an example:

image

[Dev site link](https://dev.savingsatoshi.com/en/chapters/chapter-3/coop-2?dev=true).

### PR DESCRIPTION
**Pull request by 0tuedon**
This PR closes #632 .

@afterburn , this might need an extra check because i did touch the contributions bar component.
Although everything looks fine from my end.

### CODE REVIEW

**Comment by 0tuedon** [comment]
https://github.com/saving-satoshi/saving-satoshi/assets/90271995/33b10325-100e-4699-ade3-f8855bbb7e9b

@GBKS  This is the new experience.

**Comment by benalleng** [comment]
https://saving-satoshi-git-fork-0tuedon-issue-632-656164-savingsatoshi.vercel.app/en/chapters/chapter-3/pool-2?dev=true Is probably the best page to test this on

**Comment by 0tuedon** [comment]
> https://saving-satoshi-git-fork-0tuedon-issue-632-656164-savingsatoshi.vercel.app/en/chapters/chapter-3/pool-2?dev=true Is probably the best page to test this on

Thanks ben

**Comment by GBKS** [comment]
Looks good, thank you.

If you wanted to shorten the code, you could just use `Math.max`.
```
const offsetAvatar = Math.max(cumulativeValue + playerOffset / 2 - 13, 2)
```

**Comment by 0tuedon** [comment]
> Looks good, thank you.
> 
> If you wanted to shorten the code, you could just use `Math.max`.
> 
> ```
> const offsetAvatar = Math.max(cumulativeValue + playerOffset / 2 - 13, 2)
> ```

Alright, Thanks @GBKS  will do

**Review by benalleng — APPROVED** [review]
LGTM!

File: ui/common/ContributionBar.tsx [hunk 1]
```diff
@@ -129,7 +135,7 @@ export default function ContributionBar({
           const player = protagonists[idx]
           let { offsetAvatar, offsetBar } = protagonistState[idx]
           offsetBar = offsetBar ? `${offsetBar}px` : `-100%`
-          offsetAvatar = offsetAvatar ? `${offsetAvatar}px` : `-100%`
+          offsetAvatar = offsetAvatar ? `${offsetAvatar}px` : ``
```
Comment (benalleng) [inline ui/common/ContributionBar.tsx:138]: does this affect anything in the absence of the offestAvatar now being an empty string?

Comment (ecurrencyhodler) [reply ui/common/ContributionBar.tsx:138]: I'll wait for this to be resolved b4 merging.

**Review by benalleng — APPROVED** [review]
Thanks! Now Its Good to merge @ecurrencyhodler

**Pull request merged**